validation/schema.json

```diff
@@ -132,6 +132,17 @@
                           "fixed"
                         ]
                       },
+                      {
+                        "type": "object",
+                        "properties": {
+                          "last_affected": {
+                            "type": "string"
+                          }
+                        },
+                        "required": [
+                          "last_affected"
```

Comment:
Should we also add validation to assert that a `fixed` event cannot occur alongside a `last_effected` event and vice versa?

Reply:
Thanks, that's a good idea! I'll look at doing this.

That said, I'm wondering how this would look like for GHSA specifically when an entry wants to encode both <= and < (the patched version)? Would you just be using "fixed" here to capture the < and then database_specific to capture the <= ?

Reply:
Currently if an advisory has a fix we replace the upper bound with `< f.i.x` when we rehydrate and consider it an acceptable change since. It's only for advisories that don't have a fix that we need to add the rehydration hints.

Reply:
And same thing for the other direction when transforming to OSV - if an advisory has a fix, we add the `fixed` event and don't add the rehydration hint.

Reply:
Thanks for clarifying!

Reply:
Also added the JSON validation.